src/main/java/com/gmail/yuramitryahin/controller/MovieSessionController.java

```diff
@@ -0,0 +1,55 @@
+package com.gmail.yuramitryahin.controller;
+
+import com.gmail.yuramitryahin.model.dto.MovieSessionRequestDto;
+import com.gmail.yuramitryahin.model.dto.MovieSessionResponseDto;
+import com.gmail.yuramitryahin.service.MovieSessionService;
+import com.gmail.yuramitryahin.service.mapper.MovieSessionMapper;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie-sessions")
+public class MovieSessionController {
+    private final MovieSessionService movieSessionService;
+    private final MovieSessionMapper movieSessionMapper;
+
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionMapper movieSessionMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionMapper = movieSessionMapper;
+    }
+
+    @PostMapping
+    public void add(MovieSessionRequestDto movieSessionRequestDto) {
+        movieSessionService.add(movieSessionMapper.toMovieSession(movieSessionRequestDto));
+    }
+
+    @PutMapping("/{id}")
+    public void put(@PathVariable("id") Long id, MovieSessionRequestDto movieSessionRequestDto) {
+        movieSessionService.update(movieSessionMapper.toMovieSession(id, movieSessionRequestDto));
+    }
+
+    @GetMapping("/{id}")
```

Comment:
Should we use `@GetMapping` for entity removal?

Reply:
now i found out about new annotation) fixed